app/scripts/modules/amazon/src/instance/details/utils.ts

```diff
@@ -16,7 +16,7 @@ export const applyHealthCheckInfoToTargetGroups = (
   healthMetrics.forEach(metric => {
     if (metric.type === 'TargetGroup') {
       metric.targetGroups.forEach((tg: ITargetGroup) => {
-        const group = targetGroups[tg.name];
+        const group = targetGroups[tg.name] || ({} as ITargetGroup);
```

Comment:
might as well use the fancy new hotness

```suggestion
        const group = targetGroups[tg.name] ?? ({} as ITargetGroup);
```